packages/synapse-react-client/src/components/UserCard/UserCardMedium.tsx

```diff
@@ -79,36 +88,13 @@ export const UserCardMedium: React.FC<UserCardMediumProps> = ({
   const { displayName, userName, firstName, lastName, position, company } =
     userProfile
 
-  useEffect(() => {
-    const pageClick = (_event: any) => {
-      if (!isContextMenuOpen) {
-        return
-      }
-      // hide content menu (deferred, to allow menu action to process)
-      setTimeout(() => {
-        if (isContextMenuOpen) {
-          toggleContextMenu(_event)
-        }
-      }, 10)
-    }
-    // SWC-4778: https://stackoverflow.com/questions/23821768/how-to-listen-for-click-events-that-are-outside-of-a-component
-    window.addEventListener('mouseup', pageClick, false)
-    return () => {
-      window.removeEventListener('mouseup', pageClick, false)
-    }
-  }, [])
```

Comment:
Since we are using MUI's menu (which uses Popper), we don't have to track clicks, it does it for us.